traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -825,10 +826,20 @@ func getServers(h http.Header, params map[string]string, tx *sqlx.Tx, user *auth
 		}
 
 		var joinSubQuery string
-		if err = tx.QueryRow(deliveryservice.HasRequiredCapabilitiesQuery, dsID).Scan(&dsHasRequiredCapabilities); err != nil {
+		rows, err := tx.Query(deliveryservice.GetRequiredCapabilitiesQuery, dsID)
```

Comment:
Does the `HasRequiredCapabilitiesQuery` not work here anymore? I would assume that's because of the structural changes to the table(s). So should the old query just be deleted, since we know it doesn't work?

It looks like this still returns a single row, so you could keep it shorter by continuing to use `QueryRow`.

Reply:
The `HasRequiredCapabilitiesQuery` has already been deleted.

Reply:
Oh, I didn't see that change in `deliveryservices.go`, so I guess it must've been in a different file. My bad.